firefox-ios/nimbus-features/messaging/messaging.fml.yaml

```diff
@@ -204,3 +225,12 @@ enums:
         description: The next eligible message should be shown.
       show-none:
         description: The surface should show no message.
+
+  MicrosurveyTargetFeature:
+    description: >
+      The specific feature the microsurvey is for. This is a label that matches across both Android and iOS.
+    variants:
+      printing:
+        description: The printing feature.
+      Unknown:
```

Comment:
Why do we have `Unknown`? The experiment configuration is either valid or not, no? What would this be used for and where would it present?

Reply:
I didn't want to default to the printing feature and decided to add unknown here. It seems a more safe solution in that we can handle not don't show anything at all instead of defaulting to printing and showing the wrong data. Also, this is consistent with our other enum declaration.

Happy to hear alternatives!

Note: The target feature is still TBD so `print` may change.

Reply:
The other option is to make the type Option<MicrosurveyTargetFeature> and the default null. That way, a bad config is a bad config, and you don't have to default to anything. And we'd have to handle the optional in our codebase for safety in the malformed check, so, it's kinda the same thing but cleaner?

Reply:
Would this bring any confusion that the value can be optional, but shouldn't be? While unknown is more explicit and has description for context? No strong preference, so happy to go either way.

Reply:
We did consider `null` but I agree with @cyndichin, also having a nullable enum feels like an antipattern to me